### PR TITLE
Smoll changes

### LINKED ISSUE #227: Display active auctions
Active auctions can now be retrieved via the `activeAuctions`.
There should be a new Tab `active` on the `/item` page.
When clicked it should hides the graph, show a dropdown for ordering and retrieve auctions via the new command.
To allow usage on small screens `1 Week` should be reduced to `1W` and `1 Month` to `1M`.
Also, since these are active auctions a copy button should be added to each.

### LINKED ISSUE #284: Move the copy next to the subscribe button
The copy button of the auctionDetails is basically invisible and misplaced in the lower right corner.
Especially with the recent addition to the flipper it is misplaced.
Please move it next to the subscribe button.

### LINKED ISSUE #288: Display lowest bin and seller in the flipper
Flips got extended to include the seller name and the lowest bin price for a given item.
Both should be displayed withing the flipper.
When the lowest price is clicked `/item/{itemTag}/active` (or whatever the active endpoint will end up being located at) should be opened with the filters `Bin` and `sortOrder` `lowestPrice` should be opened. 
Depends on  #227

### LINKED ISSUE #294: Fix number input filter
The PetLevel filter can't be changed to 20 easily.
The default `1` can't be deleted.

### LINKED ISSUE #296: Set the color of flips depending on the rarity
All flips contain the attribute `rarity`.
The color should be set just like on /auction
```
        COMMON = 1,
        UNCOMMON = 2,
        RARE = 3,
        EPIC = 4,
        LEGENDARY = 5,
        SPECIAL = 6,
        VERY_SPECIAL = 7,
        MYTHIC = 8,
        SUPREME = 9
```

related to #288

### PR DESCRIPTION
**Pull request by matthias-luger**
fixes #296 
fixes #294
fixes #288
fixes #284
fixes #227

### CODE REVIEW

File: src/components/ActiveAuctions/ActiveAuctions.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            }
+            setIsLoading(false);
+            setActiveAuctions(auctions);
+        }).catch(() => {
+            setIsLoading(false);
+            setActiveAuctions([]);
+        })
+    }
+
+    let recentAuctionList = activeAuctions.map((recentAuction) => {
```
Comment (Ekwav) [inline src/components/ActiveAuctions/ActiveAuctions.tsx:46]: recentAuctions?

File: src/components/ActiveAuctions/ActiveAuctions.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                            </div>
+                            <div>
+                                {numberWithThousandsSeperators(recentAuction.price)} Coins
+                            </div>
+                        </Card.Header>
+                        <Card.Body style={{ padding: "10px" }}>
+                            <img style={{ marginRight: "15px" }} crossOrigin="anonymous" className="player-head-icon" src={recentAuction.seller.iconUrl} alt="" height="24" width="24" loading="lazy" />
+                            <span>{recentAuction.playerName}</span>
+                            <hr />
+                            <p>{'ended ' + moment(recentAuction.end).fromNow()}</p>
```
Comment (Ekwav) [inline src/components/ActiveAuctions/ActiveAuctions.tsx:63]:
```suggestion
                            <p>{'ends ' + moment(recentAuction.end).fromNow()}</p>
```

File: src/components/ItemFilter/ItemFilter.tsx [hunk 3]
```diff
@@ -165,36 +127,45 @@ function ItemFilter(props: Props) {
...
+        let options = filterOptions.find(f => f.name === filterName);
+        let defaultValue: any = 0;
+        if (options && options.options[0]) {
+            defaultValue = options.options[0];
+        }
+        return (
+            <div key={filterName} className="filter-element">
+                <FilterElement onFilterChange={onFilterElementChange} options={options} defaultValue={defaultValue} />
+                <div style={{height: "100%", position: "relative"}} className="remove-filter" onClick={() => removeFilter(filterName)}>
+                    <DeleteIcon style={{marginLeft: "5px", top: "45px", position: "absolute"}} color="error" />
```
Comment (Ekwav) [inline src/components/ItemFilter/ItemFilter.tsx:157]: this overlaps with other filters if there are multiple

**Review by Ekwav — CHANGES_REQUESTED** [review]
- the filter is always set to `itemFilter=e30=` wich disabled all time
- active auctions can't be reached via direct link
- there is no sort order select on active auctions

**Comment by Ekwav** [comment]
Please also read the description of #227 you did only the thing in the title not the other 3 ;)

**Comment by matthias-luger** [comment]
the feature to remember which tab was selected should be an own issue

**Review by Ekwav — CHANGES_REQUESTED** [review]
I miss the feature to link to active auctions directly. See #288 
like using `/item/{TAG}/active`

**Comment by Ekwav** [comment]
Furthermore the copy button required by #227 is missing on active auctions

**Comment by matthias-luger** [comment]
> I miss the feature to link to active auctions directly. See #288
> like using `/item/{TAG}/active`

Like I said in my other commend, please create a seperate issue

**Comment by Ekwav** [comment]
> > I miss the feature to link to active auctions directly. See #288
> > like using `/item/{TAG}/active`
> 
> Like I said in my other commend, please create a seperate issue

Then please remove "fixes #288" as this PR does not fix it.

**Review by Ekwav — APPROVED** [review]
I hope nothing broke

File: src/utils/Formatter.tsx [hunk 4]
```diff
@@ -77,26 +77,40 @@ export function camelCaseToSentenceCase(camelCase: string): string {
...
+        { type: "MYTHIC", colourCode: "#ff55ff" },
+        { type: "SUPREME", colourCode: "#AA0000" }]
+
+
+    let color: TierColour | undefined;
+
+    if (tier) {
+        //!tier ? DEFAULT_COLOR : (TIER_COLORS[tier.toString().toUpperCase()] || 
+        if(!isNaN(Number(tier))){
+            color = tierColors[tier]
```
Comment (Ekwav) [inline src/utils/Formatter.tsx:104]: This is missing a -1

**Review by Ekwav — CHANGES_REQUESTED** [review]

**Pull request merged**